commands/new_function_test.go

```diff
@@ -423,3 +428,58 @@ func Test_getPrefixValue_Flag(t *testing.T) {
 		t.Errorf("want %s, got %s", want, val)
 	}
 }
+
+func Test_addEofNewlines_works(t *testing.T) {
```

Comment:
Where is this getting tested?

I'd expect something like if wantedYAML == os.ReadFile("stack.yml")

Reply:
Nice catch, [I just added this](https://github.com/openfaas/faas-cli/pull/902/files#diff-d5819756d14b25e2e55fd81bf9ac16c54dcb28c81b67c66da0dab67e7cc5c71cR480-R488). Let me know if anything else is needed?